modules/SofaTopologyMapping/src/SofaTopologyMapping/Edge2QuadTopologicalMapping.cpp

```diff
@@ -81,16 +81,16 @@ void Edge2QuadTopologicalMapping::init()
         msg_error() << "Radius is zero or negative";
         return;
     }
-    else if (d_radiusFocal.isSet() && d_radiusFocal.getValue() < std::numeric_limits<double>::min())
+    if (d_radiusFocal.isSet() && d_radiusFocal.getValue() < std::numeric_limits<double>::min())
     {
-        msg_error() << "Focal Radius is zero or negative";
-        return;
+        msg_warning() << "Focal Radius is zero or negative";
```

Comment:
Not sure to understand what happened now if you put a negative value. It will create a mapping?

Reply:
later on, it tests if the focal radius is positive then do something with it (otherwise it will be treated as the focal radius has not been given)